app/utils/filters.js

```diff
@@ -9,37 +9,40 @@ function validateFilters (filters) {
   var pageIsNull = !check.assigned(filters.page)
   var pageIsPositive = check.positive(Number(filters.page))
   return (pageSizeIsNull || pageSizeInRange) &&
-         (pageIsNull || pageIsPositive)
+    (pageIsNull || pageIsPositive)
 }
 
 function describeFilters (filters) {
   let description = ``
-  if (filters.fromDate) description += ` from ${filters.fromDate}`
-  if (filters.toDate) description += ` to ${filters.toDate}`
+  if (filters.fromDate) description += ` from <b>'${filters.fromDate}'</b>`
```

Comment:
Is there a reason to use `<b>` as opposed to `<strong>` or something else?

Reply:
I've always used `<b>` for bold. No other reason than that.

Reply:
`<b>` makes things bold
`<strong>` makes things bold and marks them as semantically important

Not sure which is more applicable in this case — https://www.html-5-tutorial.com/strong-and-b-elements.htm

Reply:
I tend to use strong because its likely to be important if it needs to be in bold

Reply:
I'm gonna leave it as `<b>` for now, we should bring this up next time we do an accessibility review. I'm not sure if they are more semantically important than the rest of the sentence.